sdk/sdktypes/build.go

```diff
@@ -25,13 +25,18 @@ func StrictBuildFromProto(m *BuildPB) (Build, error) { return Strict(BuildFromPr
 
 func NewBuild() Build { return kittehs.Must1(BuildFromProto(&BuildPB{})) }
 
-func (p Build) ID() (_ BuildID)      { return kittehs.Must1(ParseBuildID(p.read().BuildId)) }
-func (p Build) CreatedAt() time.Time { return p.read().CreatedAt.AsTime() }
+func (p Build) ID() (_ BuildID)          { return kittehs.Must1(ParseBuildID(p.read().BuildId)) }
+func (p Build) ProjectID() (_ ProjectID) { return kittehs.Must1(ParseProjectID(p.read().ProjectId)) }
+func (p Build) CreatedAt() time.Time     { return p.read().CreatedAt.AsTime() }
 
 func (p Build) WithNewID() Build {
 	return Build{p.forceUpdate(func(m *BuildPB) { m.BuildId = NewBuildID().String() })}
 }
 
+func (p Build) WithProjectID(pid ProjectID) Build {
```

Comment:
need to add validation of project_id to Validate() above

Reply:
done